src/main/scala/ai/privado/languageEngine/java/threatEngine/ThreatEngineExecutor.scala

```diff
@@ -102,6 +108,59 @@ class ThreatEngineExecutor(cpg: Cpg, dataflows: Map[String, Path], repoPath: Str
           case Failure(_)   => None
         }
 
+      case "Threats.CookieConsent.isCookieConsentMgmtModuleImplemented" =>
+        CookieConsentMgmtModule.getViolations(threat, cpg, dataflows, ruleCache) match {
+          case Success(res) => {
+            Some(res)
+          }
+          case Failure(_) => None
```

Comment:
In case of failure, add a log.error message denote the policy failure along with policy id